### ISSUE TITLE
Facebook Warning - Too Fast Too Furious

### ISSUE THREAD

**Issue by austinjhunt** [spieglt/fb-delete#18]
![image](https://user-images.githubusercontent.com/31296526/150663634-57a8afbd-3d54-421d-996c-a027dacfc88f.png)

Can the presence of this warning be checked for to temporarily pause execution when it happens? I'm not sure how long the waiting period needs to be though.

**Comment by spieglt** [comment]
Yep, great idea. I don't have a Facebook account anymore so not sure when I'll get around to this though. Please feel free to take a pass if you like.
